scripts/prepare/addon-bundle.ts

```diff
@@ -190,6 +192,17 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
 
   await Promise.all(tasks);
 
+  const dtsFiles = await glob(outDir + '/**/*.d.ts');
```

Comment:
Can this be moved to a utility function, given that it's the same in these two scripts? And can you add a comment explaining why this is needed (can be done in the function)